docs/indepth/installation.rst

```diff
@@ -351,9 +356,68 @@ Paste them into a root console on the target node:
+
+Both types of certificate will be signed by a certificate authority identifying your cluster, which is also generated using the ``flocker-ca`` tool.
+
+Using the machine on which you installed the ``flocker-cli`` package, run the following command to generate your cluster's root certificate authority, replacing ``mycluster`` with any name you like to uniquely identify this cluster.
+
+.. code-block:: console
+
+    $ flocker-ca initialize mycluster
+    Created cluster.key and cluster.crt. Please keep cluster.key secret, as anyone who can access it will be able to control your cluster.
+
+You will find the files ``cluster.key`` and ``cluster.crt`` have been created in your working directory.
+The file ``cluster.key`` should be kept only by the cluster administrator; it does not need to be copied anywhere.
+
+You are now able to generate authentication certificates for the control service and each of your nodes.
+To generate the control service certificate, run the following command from the same directory containing your authority certificate generated in the previous step.
+
+.. code-block:: console
+
+   $ flocker-ca create-control-certificate
+   Created control-service.crt. Copy it over to /etc/flocker/control-service.crt on your control service machine and make sure to chmod 0600 it.
+   
+You will need to copy both ``control-service.crt`` and ``control-service.key`` over to the node that is running your control service, to the directory ``/etc/flocker/``.
+You should copy these files via a secure communication medium such as SSH, SCP or SFTP.
+
+You will also need to generate authentication certificates for each of your nodes.
+Do this by running the following command as many times as you have nodes; for example, if you have two nodes in your cluster, you will need to run this command twice.
+This step should be followed for all nodes on the cluster, including the node running the control service.
+Run the command in the same directory containing the certificate authority files you generated in the first step.
+
+.. code-block:: console
+
+   $ flocker-ca create-node-certificate
+   Created 8eab4b8d-c0a2-4ce2-80aa-0709277a9a7a.crt. Copy it over to /etc/flocker/node.crt on your node machine, and make sure to chmod 0600 it.
+
+The actual certificate and key file names generated in this step will vary from the example above; when you run ``flocker-ca create-node-certificate``, a UUID for a node will be generated to uniquely identify it on the cluster and the files produced are named with that UUID.
+
+As with the control service certificate, you should securely copy the generated certificate and key file over to your node.
+Copy the generated files to ``/etc/flocker/`` on the target node and name them ``node.crt`` and ``node.key``.
+
+Finally, you should securely copy the ``cluster.crt`` file generated in the first step to ``/etc/flocker/cluster.crt`` on all of your nodes, including the machine running the control service.
```

Comment:
Putting it down here means someone who is configuring control service might not see it, and be confused when control service doesn't start.